client/rest-high-level/src/test/java/org/elasticsearch/client/MLRequestConvertersTests.java

```diff
@@ -214,12 +214,13 @@ public void testCloseJob() throws Exception {
             requestEntityToString(request));
     }
 
-    public void testDeleteExpiredData() {
-        DeleteExpiredDataRequest deleteExpiredDataRequest = new DeleteExpiredDataRequest();
+    public void testDeleteExpiredData() throws Exception {
+        DeleteExpiredDataRequest deleteExpiredDataRequest = new DeleteExpiredDataRequest(1.0f, TimeValue.timeValueHours(1));
```

Comment:
Might be better to test with values other than 1, as 1 is more likely to end up in the output due to the combination of a bug and a fluke.